core/configs/uiConfig.tsx

```diff
@@ -0,0 +1,20 @@
+type uiConfigType = {
```

Comment:
Right now we have two folders with configurations, partly due to my own doing: `/core/App/config` (application configs) and `/core/configs` (ledger configs which I probably had originally indented to be some sort of global config folder). With the introduction of the new security features, I created the `core/App/config` folder where we export configs as `.ts` files. If it's not too much work, can we stick to that convention moving forward and simply call this file `ui.ts`? I'll rename the ledger configs to be more clear.

Reply:
Yeah that makes sense to me. Should I move it to the /core/App/config directory too?

Reply:
Yeah that would be awesome!